### PR TITLE
Replace Airflow guide in docs/guides with Astro version of Marquez Airflow example

### PR DESCRIPTION
**Pull request by merobi-hub**
We need an easy-to-configure, functional Airflow quickstart, but airflow.md in docs/guides is not working currently and requires complex configuration across multiple services.

This: 

- edits and retitles airflow.md to reflect the fact that it's really a guide to the proxy (to give us a proxy guide)
- with the blessing of @wslulciuc , updates the Airflow tutorial in marquez/examples to use Astro (with the help of the Astro tutorial [here](https://docs.astronomer.io/learn/marquez)) and adds it to `guides` as `airflow-quickstart`

Note: with the changes needed to upgrade to Astro this is significantly different from the original in Marquez, so it seems that both are worth maintaining.

### CODE REVIEW

**Review by JDarDagran — APPROVED** [review]

File: docs/guides/new-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+
+<p align="center">
+  <img src={require("./docs/current-search-count.png").default} />
+</p>
+
+If you take a quick look at the lineage graph for `counter.inc`, you should see `.public.counts` as an output dataset and `sum.total` as a downstream job!
+
+![](./docs/astro-current-lineage-view-job.png)
+
+## Step 5: Troubleshoot a Failing DAG with Marquez
```
[the remark this answers is not in the document]
Comment (merobi-hub) [reply docs/guides/new-quickstart.md]: Thanks for noticing this @JDarDagran

**Comment by merobi-hub** [comment]
Thanks @kentdanas for all the improvements. Incorporating them into a single forthcoming commit.

**Review by wslulciuc — CHANGES_REQUESTED** [review]
Thanks for improving the docs, @merobi-hub. I have some suggestion to simplify the setup steps:

1. Use published Marquez image
2. Use published OpenLineage proxy image
3. Define a combined docker compose file that sets up both services; users would no longer have to clone repos to get started or run any scripts.

**Comment by merobi-hub** [comment]
Thanks @wslulciuc . Propose opening a separate PR to make the more substantial suggested improvements to the proxy guide so that the new quickstart is available. The separate PR will:
- use published images for services
- combine services in one docker compose

File: docs/guides/airflow_proxy.md [hunk 2]
```diff
@@ -1,103 +1,75 @@
...
 
-Airflow has a convenient way to setup and run a fully functional environment using [docker compose](https://docs.docker.com/compose/). The followings are therefore required to be installed before we start this tutorial.
+Airflow has a convenient way to set up and run a fully functional environment using [Docker Compose](https://docs.docker.com/compose/). The following are therefore required to be installed before we begin this tutorial.
 
 ### Prerequisites
 
 - Docker 20.10.0+
 - Docker Desktop
 - Docker Compose
+- Java 11
```
[the remark this answers is not in the document]
Comment (merobi-hub) [reply docs/guides/airflow_proxy.md]: Java requirement removed. Propose making image change in separate PR.

File: docs/guides/airflow_proxy.md [hunk 3]
```diff
@@ -1,103 +1,75 @@
...
+This tutorial introduces you to using the [OpenLineage Proxy](https://github.com/OpenLineage/OpenLineage/tree/main/proxy) with Airflow. OpenLineage has various integrations that will enable Airflow to emit OpenLineage events when using [Airflow Integrations](https://openlineage.io/docs/integrations/airflow/). In this tutorial, you will be running a local instance of Airflow using Docker Compose and learning how to enable and setup OpenLineage to emit data lineage events. The tutorial will use two backends to check the data lineage, 1) the Proxy, and 2) [Marquez](https://marquezproject.ai/).
```
[the remark this answers is not in the document]
Comment (wslulciuc) [reply docs/guides/airflow_proxy.md:7]: You can also link to the sections in markdown ;)

**Comment by merobi-hub** [comment]
Renamed `new-quickstart` as `airflow-quickstart`

**Review by wslulciuc — APPROVED** [review]
You officially have my blessing to us the Marquez example for Airflow! Great work! 💯

**Pull request merged**